### ISSUE TITLE
Locate Button HTML Tags

### ISSUE THREAD

**Issue by faridcher** [cmv/cmv-app#341]
When using "Locate Button", I get the result like the following image:
 
![snap](https://files.gitter.im/cmv/cmv-app/eetr/GPSLocation.PNG)

The widget treats the html tags as plain text.

**Comment by tmcgee** [comment]
I can confirm this is what I see as well. I believe this used to work fine though not sure when something changed to cause.

**Comment by faridcher** [comment]
You can do it easily by setting publishGPSPosition property to false. Modify the viewer.js file like this:
```javascript
locateButton: {
                include: true,
                id: 'locateButton',
                type: 'domNode',
                path: 'gis/dijit/LocateButton',
                srcNodeRef: 'locateButton',
                options: {
                    map: true,
                    publishGPSPosition: false,
                    highlightLocation: true,
                    useTracking: true,
                    geolocationOptions: {
                        maximumAge: 0,
                        timeout: 15000,
                        enableHighAccuracy: true
                    }
                }
            },
```

**Comment by jebu75** [comment]
If you like, please test the fix I just submitted in #344 and let me know if that fixes the issue for you.  Thanks!

**Comment by faridcher** [comment]
The fix is working fine. Thanks
